src/tasks/AppleAppBuilder/Templates/CMakeLists.txt.template

```diff
@@ -10,17 +10,25 @@ set(APP_RESOURCES
 add_executable(
     %ProjectName%
     %MainSource%
-    runtime.h
-    runtime.m
-    %AotModulesSource%
     ${APP_RESOURCES}
 )
 
+if(NOT %UseNativeAOTRuntime%)
+    target_sources(
+        %ProjectName%
+        PUBLIC
```

Comment:
this should be PRIVATE since we don't want to propagate the sources to anything that links with the target (doesn't really matter here but still it's the better default)

Reply:
Done.